ansible/roles/schulcloud-server-init/templates/configmap_file_init.yml.j2

```diff
@@ -298,5 +302,83 @@ data:
           }
       ]);'
     fi
-    # this is crucial, until we have all entities in nest!
+
+    # ========== Start of the Bettermarks tool configuration section.
+
+    # This is currently performed only for the following 4 namespaces:
+    #   - 'nbc-bettermarks-test',
+    #   - 'nbc-main',
+    #   - 'brb-bettermarks-test',
+    #   - 'brb-main';
+    # the first two being the testing environments for the nbc instances
+    # and the last two being the testing environments for the brb instances.
+
+    # The Bettermarks URL (entrypoint) and client secret pairs can sometimes be shared among
```

Comment:
proposition: The Bettermaks URL (entrypoint) and client secret can be shared among the different environments, but to not complicate the configuration, they are configured separately for each environment

Reply:
Good point, thanks :+1: . This sounds better, changed that.